pages/about.html

```diff
@@ -87,3 +85,34 @@ <h3 class="bio-name"></h3>
 		</div>
 	</div>
 </div>
+
+<section data-aos="fade-up" class="pb-0 mt-0" style="position:relative">
+
+	<img alt="Bolt" class="accent d-none d-md-inline" style="width:100px; position:absolute; right: 340px; top: 0px;" src="/assets/images/accents/bolt.svg" />
+	<img alt="Plus" class="accent d-none d-md-inline" style="width:130px; position:absolute; bottom: 180px; left: 0px;" src="/assets/images/accents/plus.svg" />
+	
+	<div class="container-fluid row mx-0 my-0">
+		<div class="col-0 col-md-2"></div>
+		<div class="col-12 col-md-8">
+			<h1>Our Funders</h1>
```

Comment:
Do we want to change this to "Funders and Partners" or just "Partners" or "Supporters" so we could list other orgs that have supported us from the past?

Reply:
Curious what the difference you feel is? I had "funders" in order to differentiate from "Partners" which I think might be interpreted as companies participating in the coalition. Is there a group of companies you feel should be on this list that aren't?

Reply:
Agree, think "partners" could be misinterpreted. I was thinking other entities that might have supported Impact Labs in the past like Jonathan M. Nelson Center for Entrepreneurship’s B-Labs incubator? Not super necessary, but also might broaden the number of orgs we can list (even if they don't directly fund us)